frontend/src/js/controllers/createChallengeUsingUi.js

```diff
@@ -0,0 +1,613 @@
+// Invoking IIFE for create challenge using ui
+(function() {
+
+    'use strict';
+
+    angular
+        .module('evalai')
+        .controller('CreateChallengeUsingUiCtrl', CreateChallengeUsingUiCtrl);
+
+    CreateChallengeUsingUiCtrl.$inject = ['utilities', 'loaderService', '$rootScope', '$state'];
+
+    function CreateChallengeUsingUiCtrl(utilities, loaderService, $rootScope, $state) {
+        var vm = this;
+        var userKey = utilities.getData('userKey');
+        vm.hostTeamId = utilities.getData('challengeHostTeamId');
+        vm.wrnMsg = {};
+        vm.isValid = {};
+        vm.isFormError = false;
+        vm.challengeEvalScript = null;
+        vm.challengeTitle = null;
+        vm.formError = {};
+        vm.step1 = true; // varibale to create form
+        vm.step1e = false; // variable to store edit form
+        vm.step2 = false;
+        vm.step2e = false;
+        vm.step3 = false;
+        vm.step3e = false;
+        vm.step4 = false;
+        vm.step4e = false;
+        vm.step5 = false;
+        vm.step5e = false;
+        vm.step6 = false;
```

Comment:
You can create an array or object for step variables. If possible do it so,. @RishabhJain2018